package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rubin-client",
-  "version": "0.1.0",
+  "version": "1.2.5",
```

Comment:
there's a version number for published releases so it seems like it would make sense to put it in the package.json too, right?